### DISCUSSION TITLE
oracledb.exceptions.NotSupportedError: DPY-3012: national character set id 871 is not supported by python-oracledb in thin mode

### DISCUSSION THREAD

**Discussion by nchillal** [oracle/python-oracledb#28] [General]
Getting below error `oracledb.exceptions.NotSupportedError: DPY-3012: national character set id 871 is not supported by python-oracledb in thin mode` after installing python-oracledb 

Database is running in utf-8 and it can't be changed. How can this be resolved?

**Comment by anthony-tuininga** [comment]
See #16. I pushed a patch three days ago that allows you to connect, but not fetch data in NCHAR, NVARCHAR2 or NCLOB columns. If that restriction is acceptable and you are able to build binaries for yourself, then you can use that approach. If those restrictions are unacceptable, you'll have to enable thick mode. See the [documentation](https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enablingthick) for details!

**Reply by anthony-tuininga** [reply]
This patch was included in python-oracledb 1.0.2 which was just released!

**Reply by stdrone** [reply]
Hello!
Accordinally to documentation with not supported NLS we cannot fetch data from NCLOB. But this limitation aplies for CLOB too.
Is it can be fixed?
